models/biocro/R/model2netcdf.BIOCRO.R

```diff
@@ -77,6 +77,7 @@ model2netcdf.BIOCRO <- function(result, genus = NULL, outdir, lat = -9999, lon =
       TotLivBiom = k * (Leaf + Root + Stem + Rhizome + Grain), 
       RootBiom = k * Root, 
       StemBiom = k * Stem, 
+      Yield = Stem,
```

Comment:
Perhaps we should leave 'yield' out completely, as the components of yield vary by crop (e.g. maize --> grain, willow --> stem, biomass grass -->  stem + leaf